### PR TITLE
Add $args to eklipse

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

Commands was missing $args, so I have added that and I have also updated the tests. 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/eklipse/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,36 @@
+nextflow_process {
+
+    name "Test Process EKLIPSE"
+    script "modules/nf-core/eklipse/main.nf"
```
Comment (sateeshperi) [inline modules/nf-core/eklipse/tests/main.nf.test:4]:
```suggestion
    script "../main.nf"
```

File: modules/nf-core/eklipse/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    process "EKLIPSE"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "eklipse"
+
+    test("eklipse") {
```
Comment (sateeshperi) [inline modules/nf-core/eklipse/tests/main.nf.test:10]:
```suggestion
    test("homo_sapiens [bam]") {
```

**Comment by sateeshperi** [comment]
Its ok that you deleted pytests for this module....we planned to delete them in bulk after the migration....either way is fine

**Review by jemten — APPROVED** [review]
Looks good!

**Pull request merged**